training/distributed_training/pytorch/model_parallel/gpt-neox/smp-train-gpt-neox-sharded-data-parallel.ipynb

```diff
@@ -0,0 +1,924 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #42.        checkpoint_local_path=hyperparameters["checkpoint-dir"] if use_fsx else None,
</pre><p> </p><p><code>checkpoint_local_path</code> is to specify the local path in the training container. I'm not sure if this is a correct way to set the path for checkpointing to FSx.</p><p> </p><p>Ref: https://sagemaker.readthedocs.io/en/stable/api/training/estimators.html#sagemaker.estimator.Framework</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3991/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>

Reply:
<p>nevermind! I checked the parameter and it's set to a local path.</p>